pkgs/by-name/su/supercell-wx/package.nix

```
@@ -0,0 +1,154 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  fetchpatch,
+  aws-sdk-cpp,
+  bzip2,
+  cmake,
+  ninja,
+  zlib,
+  openssl,
+  curl,
+  glew,
+  geos,
+  boost185,
```

```suggestion
  boost,
```
boost 1.85 was removed some time ago. New default version is 1.86 so it should work. Also replace below.

---

I've made the suggested change.